upgrading/upgrade-roxctl.adoc

```diff
@@ -17,6 +17,12 @@ To upgrade {product-title} to the latest version, you must perform the following
 * Upgrade Scanner
 * Verify that all secured clusters are upgraded
 
+[IMPORTANT]
+====
+* In order to ensure optimal functionality it is recommended to keep your secured clusters up to date and on the same version as Central. Out of date Sensor versions are supported for 6 months, but may not have the full functionality of an up to date version.
```

Comment:
There is an `[IMPORTANT]` block below in the `== Upgrading all secured clusters` section. I suggest adding this as a last bullet point there. Something like
```
* If you are not using automatic upgrades, you must run the instructions in this section on all secured clusters including the Central cluster. <- this item already exists
** In order to ensure optimal functionality it is recommended to keep your secured clusters on the same version as Central. <- this is our new item
```

Reply:
I've moved it over to that block.